css/store.css

```diff
@@ -0,0 +1,247 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  color: black;
+}
+
+a:hover {
+  color: orange;
+}
+
+a,
+li {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+  color: inherit;
+}
+
+button {
+  padding: 20px;
+  border: 0px;
+  background: none;
+  font-weight: 100;
+}
+
+hr {
+  display: block;
+  margin-top: 40;
+  margin-bottom: 0.5em;
+  margin-left: 100px;
```

Comment:
rem are usually more responsive than pixels, you can read more https://chiamakaikeanyi.dev/sizing-in-css-px-vs-em-vs-rem/